tests/plugins_test.go

```diff
@@ -58,6 +58,12 @@ exec docker run --log-driver=none --rm -i {{.ImageName}}:{{.Version}} "$@"
 		"buf.build/community/neoeinstein-prost-serde": {"no_include"},
 		"buf.build/community/neoeinstein-tonic":       {"no_include"},
 	}
+	// Some plugins do not generate any code for the test protos, so we allow an empty plugin.sum file for these
+	// plugins.
+	allowedEmptyPluginSums = map[string]bool{
+		"buf.build/bufbuild/validate-java": true,
+		"buf.build/grpc-ecosystem/gateway": true,
```

Comment:
Do we want to add an exception for the`buf.build/community/mercari-grpc-federation` plugin for now until a fix is made available (at least for the current version)?

Reply:
I want to undo and remove that plugin entirely, and re-open the issue advocating for an upstream fix. Any concerns with that?